Remove data file installation from setup.py

Manual handling is required due to the inflexibility of distutils.

diff --git a/setup.py b/setup.py
index 19d4195..33a5601 100644
--- a/setup.py
+++ b/setup.py
@@ -48,11 +48,4 @@ setup(
             'autosuspend = autosuspend:main',
         ]
     },
-
-    data_files=[
-        ('etc', ['data/autosuspend.conf',
-                 'data/autosuspend-logging.conf']),
-        ('lib/systemd/system', ['data/autosuspend.service',
-                                'data/autosuspend-detect-suspend.service']),
-    ],
 )
